### PR TITLE
Update for gcp and social

### PR DESCRIPTION
**Pull request by raymondjacobson**

### CODE REVIEW

File: nginx/nginx.template [hunk 1]
```diff
@@ -90,6 +90,11 @@ http {
       proxy_pass ${BEDTIME_URL};
     }
 
+    # Proxy social
+    location /social/ {
+      proxy_pass http://social:8001/;
```
Comment (isaacsolo) [inline nginx/nginx.template:95]: maybe remove trailing `/`

Comment (isaacsolo) [reply nginx/nginx.template:95]: for both location and after the port number

**Review by isaacsolo — APPROVED** [review]

File: nginx/nginx.template (hunk 1 above)
Comment (raymondjacobson) [reply nginx/nginx.template:95]: unfortunately this is necessary -- it's how nginx strips /social/ from the forwarded url

Comment (raymondjacobson) [reply nginx/nginx.template:95]: https://stackoverflow.com/questions/41605137/how-to-remove-a-subdirectory-from-the-url-before-passing-to-a-script

Comment (raymondjacobson) [reply nginx/nginx.template:95]: better one https://serverfault.com/questions/699695/nginx-proxy-pass-remove-url-substring

**Pull request merged**